spec/helpers/comments_helper_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the CommentsHelper. For example:
+#
+# describe CommentsHelper do
```

Comment:
Is this a default test?

Reply:
Yeah this was created when we used rails generate